BE/src/main/java/team01/issuetracker/controller/AssigneeController.java

```diff
@@ -0,0 +1,25 @@
+package team01.issuetracker.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import team01.issuetracker.service.AssigneeService;
+
+@RequiredArgsConstructor
+@RequestMapping("/api/assignees")
+@RestController
+public class AssigneeController {
+
+    private final AssigneeService assigneeService;
+    private final Logger logger = LoggerFactory.getLogger(AssigneeController.class);
```

Comment:
저 같은 경우 필드 순서는 아래와 같이 배치하는 편이에요.

```suggestion
    private final Logger logger = LoggerFactory.getLogger(AssigneeController.class);
    private final AssigneeService assigneeService;
```

`@Slf4j` 어노테이션을 사용하셔도 됩니다.